Projects/TEAM-1/Mykola Kukla/206/206/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+// 5. Введіть три числа і виведіть на екран значення суми і добуток цих чисел.
```

Comment:
Please, check your requirements and the actual code proposes. Here are some mismatches.